test/battle/ability/gale_wings.c

```diff
@@ -50,4 +50,43 @@ SINGLE_BATTLE_TEST("Gale Wings only grants priority to Flying-type moves")
     }
 }
 
-TO_DO_BATTLE_TEST("Gale Wings doesn't increase priority of Flying-type Hidden Power, Natural Gift, Judgment or Tera Blast");
+SINGLE_BATTLE_TEST("Gale Wings doesn't increase priority of Flying-type Natural Gift, Judgment, Hidden Power, or Tera Blast")
+{
+    u32 move;
+    u16 heldItem;
+    PARAMETRIZE { move = MOVE_NATURAL_GIFT; heldItem = ITEM_LUM_BERRY; }
+    PARAMETRIZE { move = MOVE_JUDGMENT; heldItem = ITEM_SKY_PLATE; }
+    PARAMETRIZE { move = MOVE_HIDDEN_POWER; heldItem = ITEM_NONE; }
+    GIVEN {
+        ASSUME(B_GALE_WINGS >= GEN_7);
+        ASSUME(gMovesInfo[MOVE_NATURAL_GIFT].effect == EFFECT_NATURAL_GIFT);
+        ASSUME(gMovesInfo[MOVE_JUDGMENT].effect == EFFECT_CHANGE_TYPE_ON_ITEM);
+        // IV combinations sourced from https://www.smogon.com/forums/threads/hidden-power-iv-combinations.78083/
+        ASSUME(gMovesInfo[MOVE_HIDDEN_POWER].effect == EFFECT_HIDDEN_POWER);
+        ASSUME(gMovesInfo[MOVE_TERA_BLAST].effect == EFFECT_TERA_BLAST);
+        ASSUME(gItemsInfo[ITEM_SKY_PLATE].holdEffect == HOLD_EFFECT_PLATE);
+        ASSUME(gItemsInfo[ITEM_SKY_PLATE].secondaryId == TYPE_FLYING);
+        ASSUME(gNaturalGiftTable[ITEM_TO_BERRY(ITEM_LUM_BERRY)].type == TYPE_FLYING);
+        OPPONENT(SPECIES_TALONFLAME) { Ability(ABILITY_GALE_WINGS); Speed(1); Item(heldItem); HPIV(31); AttackIV(3); DefenseIV(31); SpAttackIV(30); SpDefenseIV(30); SpeedIV(30); TeraType(TYPE_FLYING); }
+        PLAYER(SPECIES_WOBBUFFET) { Speed(100); };
+    } WHEN {
+        TURN { MOVE(opponent, move); }
+    } SCENE {
+        if (move == MOVE_NATURAL_GIFT) {
+            MESSAGE("Wobbuffet used Celebrate!");
+            MESSAGE("Foe Talonflame used Natural Gift!");
+        }
+        else if (move == MOVE_JUDGMENT) {
+            MESSAGE("Wobbuffet used Celebrate!");
+            MESSAGE("Foe Talonflame used Judgment!");
+        }
+        else if (move == MOVE_HIDDEN_POWER) {
+            MESSAGE("Wobbuffet used Celebrate!");
+            MESSAGE("Foe Talonflame used Hidden Power!");
+        }
+        else {
+            MESSAGE("Wobbuffet used Celebrate!");
+            MESSAGE("Foe Talonflame used Tera Blast!");
```

Comment:
If the Celebrate message gets used either way, it'd be easier to put it outside of the conditions and only have the "Foe Talonflame used X" depend on the move

Reply:
I'm checking for both messages specifically because the order matters, Wobbuffets message needs to be seen first for the test to evaluate speed properly as far as I'm aware :)